### DISCUSSION TITLE
Create auto-importers from other popular text expanders

### DISCUSSION THREAD

**Discussion by federico-terzi** [espanso/espanso#1232] [Feature requests and Ideas]
It would be nice to have an automatic way to import snippets from other popular text expanders. That would make migration easier for a lot of users

**Comment by Kmfernan5** [comment]
Can we import from PhraseExpander?

**Comment by federico-terzi** [comment]
@Kmfernan5 Not currently, but we might implement it in the future :)

**Comment by Kmfernan5** [comment]
Can we import from CSV?

**Comment by Kmfernan5** [comment]
I tried this but it didn't work.

https://gist.github.com/mturilin/699a50d0433010708350b8befe4f97a2

PS C:\Users\***> python
Python 3.10.1 (tags/v3.10.1:2cd268a, Dec 6 2021, 19:10:37) [MSC v.1929 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> convert.py
Traceback (most recent call last):
File "<stdin>", line 1, in <module>
NameError: name 'convert' is not defined

**Comment by krystof-k** [comment]
Possibly also exports and using Property list (.plist) format, see https://github.com/espanso/espanso/discussions/1314.

**Comment by smeech** [comment]
See https://ee.qqv.com.au/usage/convert/
